src/setupWorker/glossary.ts

```diff
@@ -54,6 +59,14 @@ export type StartOptions = SharedOptions & {
    * instance is ready. Defaults to `true`.
    */
   waitUntilReady?: boolean
+
+  /**
+   * A predicate function that allows you override the default strict matching behavior.
+   * This is primarily useful in the scenario that you are using proxies you are using proxies
+   * and having difficulty getting the origin and scriptURL to match due to port differences.
+   * Default: (scriptURL, absoluteWorkerUrl) => worker.scriptURL ==== absoluteWorkerUrl
+   */
+  serviceWorkerMatcher?: ServiceWorkerMatcher
```

Comment:
What do you think if we call this method `findWorker`?

What it does is supplies the function to the `.find()`, and looks up a given worker script URL in the list of all registered workers (incl. third-party workers).

Reply:
Agreed and updated